installation/scripts/generate-kyma-installer.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+KYMA_PATH="${CURRENT_DIR}/../.."
+INSTALLER_YAML_PATH="${KYMA_PATH}/installation/resources/installer.yaml"
+UI_TEST_SECRET_PATH="${KYMA_PATH}/installation/resources/ui-test-secret.yaml.tpl"
+
+VERSIONS_ENV_PATH=""
+INSTALLER_CONFIG=""
+OUTPUT_PATH=""
+
+if [[ ! $# -eq 2 ]] ; then
+    echo 'Provide correct input arguments'
+    exit 1
+fi
+
+case "$1" in
+    "local")   
+        INSTALLER_CONFIG="${KYMA_PATH}/installation/resources/installer-config-local.yaml.tpl"
+        OUTPUT_PATH="${KYMA_PATH}/kyma-installer/local-kyma-installer.yaml" ;;
+    "cluster") 
+        INSTALLER_CONFIG="${KYMA_PATH}/installation/resources/installer-config-cluster.yaml.tpl"
+        OUTPUT_PATH="${KYMA_PATH}/kyma-installer/cluster-kyma-installer.yaml" ;;    
+    *) 
+        echo 'Provide which installer you want to generate by passing as argument "local" or "cluster"'
```

Comment:
Add "usage" function that explains usage and returns error code 1. Refer to this function anytime args are incorrect.